packages/koa-shopify-auth/src/auth/create-oauth-callback.ts

```diff
@@ -53,11 +53,15 @@ export default function createOAuthCallback(config: AuthConfig) {
     }
 
     const accessTokenData = await accessTokenResponse.json();
-    const {access_token: accessToken} = accessTokenData;
-
+    const {access_token: accessToken,
+      associated_user_scope: associatedUserScope,
+      associated_user: associatedUser} = accessTokenData;
```

Comment:
```suggestion
    const {
      access_token: accessToken,
      associated_user_scope: associatedUserScope,
      associated_user: associatedUser
    } = accessTokenData;
```
I think prettier is looking for something like this. I am not sure why it is only failing on a single version though.

Reply:
Still fails